layouts/_default/single.html

```diff
@@ -2,9 +2,7 @@
 	<div class="row flex-xl-nowrap">
 		{{ if (in site.Params.doks.sectionNav .Section) -}}
 		<div class="col-lg-5 col-xl-4 docs-sidebar{{ if ne site.Params.doks.navbarSticky true }} docs-sidebar-top{{ end }}{{ if site.Params.doks.headerBar }} docs-sidebar-offset{{ end }} d-none d-lg-block">
-			<nav {{ if eq site.Params.doks.menu.section.collapsibleSidebar false }}id="sidebar-default" {{ end }}class="docs-links" aria-label="Main navigation">
-				{{ partial "sidebar/docs-menu.html" . }}
-			</nav>
+			{{ partial "sidebar/section-menu.html" . }}
```

Comment:
Maybe I don't have understand but this parameter is not used `site.Params.doks.menu.section.collapsibleSidebar`?

Reply:
Correct. Now, when no `sidebar_<section>` menu entries are found, Doks automatically generates a sidebar based on the filesystem structure of your documentation — no parameter needed. The new setup is a lot smarter and more robust.

Reply:
Thanks for your response, I don't understand how can I get the same design like this before this commit :

![image](https://github.com/gethyas/doks-core/assets/1135513/3b51e6b3-61fd-49e0-98ce-3c5fa862e322)

not like this (last version)

![image](https://github.com/gethyas/doks-core/assets/1135513/f21664e4-33b2-4f30-b99c-740d1f3194c8)

I don't want colapse menu ;)